guides/line-management/probation.md

```diff
@@ -0,0 +1,89 @@
+# Probation at Made Tech
+
+When someone joins Made Tech, they're "on probation" for a short time.
+This means their notice period is shorter - usually one week.
+
+At the end of probation, people either pass or fail. Most people pass.
+Rarely, you'll extend probation, or pass someone early.
+
+This guidance is written for line managers, but may be useful for people on probation too.
+
+- [What happens during probation?](#what-happens-during-probation)
+- [Explaining how to pass probation](#explaining-how-to-pass-probation)
+- [One-to-one meetings during probation](#one-to-one-meetings-during-probation)
+- [Finishing probation](#finishing-probation)
+
+## What happens during probation?
+
+In the first week, explain probation to your direct report. Talk about what it is, and how to pass it.
+
+Each week, meet them one-to-one.
+Get feedback from their team, and reflect on it together.
+Talk often about whether they're on-track to pass.
+
+At the end of probation, pass or fail it, and explain what happens next.
+
+## Explaining how to pass probation
+
+Talk about what good looks like for your direct report's role. [The handbook role pages](../../roles/README.md) may help with this.
```

Comment:
I think this should link to `roles/sfia` - the core competencies and behaviours described there are IMO the most useful reference when trying to determine what being successful in that role will look like

Reply:
I think SFIA without context is scary - linking to `roles/sfia` gives you a list of docs rather than a formatted README.

I see it as the job of the roles "landing page" to surface whatever is most important - if that's SFIA, sure, but I'd change that page, not this link.

Reply:
You could start [here](https://github.com/madetech/handbook/blob/master/roles/sfia/academy_software_engineer.md), but maybe an index page would be better. The pages themselves are good.

Reply:
Or there is this: https://github.com/madetech/handbook/blob/master/roles/README.md#technology

Reply:
That roles page that you've linked second is where this ends up - without the #technology anchor tag, since probation period is across non-tech roles too (as far as I know!)